### PR TITLE
black

### PR DESCRIPTION
**Pull request by CJ-Wright**

### CODE REVIEW

**Comment by beckermr** [comment]
We should add black checking to the tests if we have not already.

**Comment by CJ-Wright** [comment]
This PR has that

File: .travis.yml [hunk 1]
```diff
@@ -27,6 +27,7 @@ script:
   - coverage run run_tests.py
   - coverage report -m
   - codecov -X gcov
+  - black --check --verbose --diff --config=pyproject.toml .
```
Comment (beckermr) [inline .travis.yml:30]: Ack nvm!

Comment (CJ-Wright) [inline .travis.yml:30]: @beckermr I copied this out of conda-smithy

**Pull request merged**